com.baidu.NetDisk.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.baidu.NetDisk
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+command: baidunetdisk
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-download
+  - --filesystem=xdg-config/baidunetdisk:create
```

Comment:
are you sure you need this?

Reply:
which is why the build fails.

Reply:
Unfortunately, this seems to be needed. Part of their logic appears to hardcode `~/.config/baidunetdisk` and insists on putting files there.

I didn't use `--persist`, because as an online storage service client (that doesn't use XDG Portal), some users might want to give it access to `home`, in which case `--persist=.config/baidunetdisk` would break. 

Personally (and judging from the quality of this software), I wouldn't trust it with read access to my entire home directory, but I want to leave that possibility open for people that need more convenience, without introducing too much hassle.

Reply:
use --persist